src/09-reveal/editor.scss

```diff
@@ -0,0 +1,22 @@
+@charset "UTF-8";
+/*********************************************************************************************
+
+Plugin Name: MOJ Blocks
+Block: Reveal (Gutenberg editor styles)
+Text Domain: mojblocks
+
+**********************************************************************************************/
+
+@import 'style.scss';
+
+.govuk-details[open]>.govuk-details__summary:before {
+  display: block;
+  width: 0;
+  height: 0;
+  -webkit-clip-path: polygon(0 0,50% 100%,100% 0);
+  clip-path: polygon(0 0,50% 100%,100% 0);
+  border-color: transparent;
+  border-style: solid;
+  border-width: 12.124px 7px 0;
+  border-top-color: inherit;
+}
```

Comment:
Morning :) I posted about EditorConfig to assist in reaching our minimum standards in wordpress-gang earlier this week. Did you see this?

In fact, this stylesheet is formatted differently to the one above (hero/style.scss) - which standard are you using?

Reply:
Styles format updated.